### ISSUE TITLE
Firebase 5.2.1 Assemblies Not Loading in Unity 2018.3.0b1

### ISSUE THREAD

**Issue by BenSteenson** [firebase/quickstart-unity#212]
I am receiving the following errors when upgrading a Unity 2018.2.7f1 project with Firebase 5.2.1 (Analytics only) installed to the newest beta, Unity 2018.3.0b1, on every refresh:

```
Assembly 'Assets/Firebase/Editor/Firebase.Editor.dll' will not be loaded due to errors:
Unable to resolve reference 'Unity.iOS.Extensions.Xcode'. Is the assembly missing or incompatible with the current platform?

Assembly 'Assets/PlayServicesResolver/Editor/Google.IOSResolver.dll' will not be loaded due to errors:
Unable to resolve reference 'Unity.iOS.Extensions.Xcode'. Is the assembly missing or incompatible with the current platform?
```

Unity Player Settings:

```
Platform: iOS
Run in Xcode as: Release
Development Build: False
Scripting Runtime Version: .NET 4.x Equivalent
Api Compatibility Level: .NET 4.x
Architecture: Universal
Strip Engine Code: True
Managed Stripping Level: Normal
```

I have also installed IncrementalCompiler Version 0.0.42 (Preview 19), and my csproj files are set to use C# Language Level C# 7.3. All non-Firebase script and plugin folders are using asmdef files.

As this is only related to Editor DLLs (the iOS Play Resolver and the Firebase Editor), I have not tested on device yet.

![image](https://user-images.githubusercontent.com/40196421/45405806-ae9b3100-b6a7-11e8-95ae-283eea7b0bc8.png)

**Comment by nelsonlarocca** [comment]
That happened to me but just AFTER upgrading Xcode to version 10

**Comment by danielpunct** [comment]
Still happening with firebase 5.3.0. Anybody found a fix?

**Comment by stewartmiles** [comment]
This should be fixed in the 5.3.1 SDK release https://firebase.google.com/support/release-notes/unity#5.3.1 .  Though I noticed that Unity 2018.3 beta enables .NET 4.6 by default which means you'll need to do https://firebase.google.com/docs/unity/setup#installing_plugins_in_unity_2017x_net_4x_projects
